docs/source/nlp/machine_translation.rst

```diff
@@ -414,6 +414,27 @@ the checkpoints that had the top 5 (by default) sacreBLEU scores.
 
 At the end of training, a ``.nemo`` file is written to the result directory which allows to run inference on a test set.
 
+Multi-Validation
```

Comment:
Although its kind of obvious from the name, it's probably best to state that this works only with `model.validation_ds`.

Reply:
Updated it.